### DISCUSSION TITLE
Users!

### DISCUSSION THREAD

**Discussion by QR2020** [open-webui/open-webui#1962] [Q&A]
Is it possible for users to text each other? Not to the Models. Thank you!

**Comment by tjbck** [comment]
I was just thinking of experimenting with this concept, one idea I have is a group discussion feature with many users within one chat. I'm open to suggestions also!

**Reply by QR2020** [reply]
Great idea! Admin can create group chats, and users can chat with each other. At the same time, the admin can set the default model for summarizing and concluding chat histories. Thank you!

**Reply by nijisakai** [reply]
Also the model can appear like a normal user, so users of the group can @ this robot to do things.

**Reply by mistertwo** [reply]
What about having openweb-ui hook into an external chat system (may solve some of the UI issues), and have the LLM configuration options (tools, prompts, etc.) reside within openweb-ui, turning it into an LLM proxy of sorts? 

Or is the intent to keep all of this functionality within openweb-ui?

**Comment by andreafuturi** [comment]
My idea is:

> - We can tag users in any chat like we currently do for models/agents
> - If a user is tagged the chat is automatically shared to him and he can partecipate in the thread.
> - If more than a user is present in the chat, AI will start replying only when consulted (tagged) or even better:
If a model thinks its important to say something it will otherwise it won't say anything 


What do you think?

**Reply by andreafuturi** [reply]
And in the top left drop-down where we select models we can also search and click one of the users so that we can directly start a chat with them

**Comment by flefevre** [comment]
This feature will be great. Enabling team members to work together.
By team members it could be human but also bits based upon a model, doc, prompt, and tools .

The human member could select several discussions and ask to summarise it to put it in memory and accessible to futur rag research.

**Comment by lee-b** [comment]
I think the best way to do this would be to make the workspaces projects, with files, and have the multiple chats be within the context of a workspace, so that multiple users (humans/AI's, arbitrarily) can chat separately in subteams/working groups about their role in the workspace.  For example, you could create a workspace for a project, create a folder for docs, have an agent who writes docs working with the project manager and architect and compliance officer, and another folder for marketing materials with sales, marketing, and the docs team involved, etc., whilst project managers and product managers get oversight and input into all of the folders and a member from each team, or something.  Tools could be added to the workspace or particular agents to edit files, review them, etc., whilst other agents must interact through discussion and requests/reports.

**Reply by tjbck** [reply]
Great Idea! Related: https://github.com/open-webui/open-webui/issues/2058

**Comment by nithin8702** [comment]
This idea looks interesting. When shall we except this work to be released?

**Reply by DrShadow34** [reply]
Never apparently.

**Reply by bess-trailer** [reply]
I've been patiently waiting for months for this functionality to roll out in V0.6.0, i guess we just give up at this point.

**Reply by flefevre** [reply]
You have already since version 0.5.2 channels to enable discussion between users.
So for me this feature is already deployed

**Reply by DrShadow34** [reply]
Actually, yes, that exactly what OP requesting in initial message. You just need to turn it on in admin panel > Channels (Beta). Thank you.
Alto, there no ability to add model as member of the channel, what I'm interested in. Had to write whole damn telegram bot just for that.
